.gitattributes

```diff
@@ -0,0 +1,22 @@
+# Auto detect text files and perform LF normalization
```

Comment:
I also think there was some work on cargofying example in another PR. Should do the same.

Finally, we should consider replacing some of ll with winapi.